### PR TITLE
feat:adding address page

### PR DESCRIPTION
**Pull request by mariyasada**
adding address management functionality. you can add a new address.
add address management link to a user profile page

https://user-images.githubusercontent.com/77275100/153715337-cd397855-7735-4622-b527-8bffcd0ab665.mp4
address

### CODE REVIEW

**Review by Bhakti199 — APPROVED** [review]
looks cool, good to merge

File: stylesheets/address/address.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.add-address-container
+{
+    height: 700px;
```
Comment (ShreyashMahajan) [inline stylesheets/address/address.css:3]: you can use rem fo more than 1rem

**Review by ShreyashMahajan — APPROVED** [review]
You can  merge

File: stylesheets/address/address.css (hunk 1 above)
Comment (mariyasada) [reply stylesheets/address/address.css:3]: i knowingly gave pixel here

**Pull request merged**